<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>941</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SchoolMap  -ver1.16           -code by : jannchie</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;
</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>864</width>
     <height>786</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="ordinateInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>880</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>311</width>
      <height>177</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>搜索想要前往的地方：</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="searchLine"/>
       </item>
       <item>
        <widget class="QPushButton" name="searchPoint">
         <property name="text">
          <string>搜索</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="accessibleName">
        <string>2</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>请选择一个地点：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="chooseAddress">
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>南大门</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>宋健楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>主楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>图书馆</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>M楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>G楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>大学生服务区</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>体育场</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>哈工大门诊</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>校史馆</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新图书馆</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>大学生活动中心/训练场馆</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>学子餐厅</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>旧篮球场</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新篮球场</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>实验楼C/E</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>留学生公寓/留学生餐厅</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>教工住宅区</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>西南门</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>教工住宅高层</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B楼</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>学苑餐厅</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>一公寓/二公寓/三公寓/四公寓/五公寓</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>六公寓</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>七公寓</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>八公寓/九公寓/十公寓/十一公寓</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>北门</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>东门</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="getStart">
         <property name="text">
          <string>设为起点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="getEnd">
         <property name="text">
          <string>设为终点</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="searchWay">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>开始寻路</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="currentpoint">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>880</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new/map/icon/currentpoint.ico&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="point">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>850</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new/map/icon/point_24.ico&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="endpoint">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>860</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new/map/icon/point_24.ico&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="PaintLabel" name="mapdraw">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>921</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="mapview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>901</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new/map/map.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>220</y>
      <width>311</width>
      <height>691</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="opaqueResize">
     <bool>true</bool>
    </property>
    <property name="handleWidth">
     <number>10</number>
    </property>
    <property name="childrenCollapsible">
     <bool>true</bool>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="toolTipDuration">
      <number>1</number>
     </property>
     <property name="styleSheet">
      <string notr="true">splitter{background-color: rgb(255, 255, 255);}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <property name="handleWidth">
      <number>10</number>
     </property>
     <property name="childrenCollapsible">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="info">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>地点简介：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>活动一览表：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchActivity"/>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>1223</height>
          </size>
         </property>
         <property name="autoScroll">
          <bool>false</bool>
         </property>
         <property name="autoScrollMargin">
          <number>12</number>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::CustomDashLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>85</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>20</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>1</number>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>活动名称</string>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="foreground">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>日期</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>地点</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>目录</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="addActivity">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>添加活动</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteActivity">
           <property name="text">
            <string>删除选中的活动</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>我关注的活动：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>1223</height>
         </size>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
        <property name="autoScroll">
         <bool>false</bool>
        </property>
        <property name="autoScrollMargin">
         <number>12</number>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::CustomDashLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>85</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>1</number>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>活动名称</string>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="foreground">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </column>
        <column>
         <property name="text">
          <string>日期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>时间</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>地点</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>目录</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="searchfocusway">
          <property name="text">
           <string>以当前顺序寻路</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteFocusActivity">
          <property name="text">
           <string>删除所选已关注的活动</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QToolButton" name="close">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
background-color: rgb(225, 225, 225);
color: rgb(0, 0, 0);
border: 1px solid rgb(150, 150, 155);
}

QToolButton:hover{
background-color: #CB4042;
color: rgb(255, 255, 255);
border: 1px solid rgb(124, 124, 124);
}

QToolButton:pressed{
background-color: rgb(166, 64, 66);
color: rgb(255, 255, 255);
border: 1px solid rgb(124, 124, 124);
}

</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QToolButton" name="login">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Login in..</string>
    </property>
   </widget>
   <zorder>splitter_2</zorder>
   <zorder>mapview</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>mapdraw</zorder>
   <zorder>point</zorder>
   <zorder>endpoint</zorder>
   <zorder>currentpoint</zorder>
   <zorder>ordinateInfo</zorder>
   <zorder>close</zorder>
   <zorder>login</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QStatusBar{rgb(22, 26, 59)}</string>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDate</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaintLabel</class>
   <extends>QLabel</extends>
   <header>paintlabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchLine</tabstop>
  <tabstop>searchPoint</tabstop>
  <tabstop>chooseAddress</tabstop>
  <tabstop>getStart</tabstop>
  <tabstop>getEnd</tabstop>
  <tabstop>searchWay</tabstop>
  <tabstop>searchActivity</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>addActivity</tabstop>
  <tabstop>deleteActivity</tabstop>
  <tabstop>tableWidget_2</tabstop>
  <tabstop>searchfocusway</tabstop>
  <tabstop>deleteFocusActivity</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
